src/main/scala/io/iohk/ethereum/network/rlpx/RLPxConnectionHandler.scala

```diff
@@ -137,11 +137,8 @@ class RLPxConnectionHandler(
       case Success(message) =>
         context.parent ! MessageReceived(message)
 
-      case Failure(ex) => if (log.isDebugEnabled) {
-        log.error(ex, s"Cannot decode message from $peerId")
-      } else {
-        log.error(s"Cannot decode message from $peerId because ${ex.getMessage}")
-      }
+      case Failure(ex) =>
+        log.debug(s"Cannot decode message from $peerId, because of ${ex.getMessage}")
```

Comment:
really, will it? In fact I only changed error -> debug (and removed log.isDebugEnabled as it's not needed)